### ISSUE TITLE
countryCodeEditable set to false results in error

### ISSUE THREAD

**Issue by johngarcia9110** [bl00mber/react-phone-input-2#159]
Setting the countryCodeEditable option to false results in the following error in console:
```
Uncaught TypeError: Cannot read property 'dialCode' of undefined
```

Here is my implementation:
```
<PhoneInput
    country={'us'}
    preferredCountries={['us']}
    value={phone}
    countryCodeEditable={false}
    onChange={handlePhoneInputChange}
    onBlur={(e) => handlePhoneInputBlur(e.target.value)}
/>
```

This behavior can also be duplicated on the repo [demo page](https://bl00mber.github.io/react-phone-input-2.html), by entering {"country":"us", "countryCodeEditable":false} into the textarea.

**Comment by bl00mber** [comment]
Yeah this happens when I change demo and re-render it but not if I put this initially. Strange behavior that needs to be inspected.

**Comment by bstrech** [comment]
I can reproduce reliably without needing to set country value. 

- Configure countryCodeEditable = false
- Render component in a browser, select a country i.e. Japan, add a number
- Select a different country, then attempt to remove the value, or select all in the input field

result is the exception:
Uncaught TypeError: Cannot read property 'dialCode' of undefined
    at r.handleInput (lib.js:6)
    at HTMLUnknownElement.callCallback (react-dom.development.js:149)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:199)
    at invokeGuardedCallback (react-dom.development.js:256)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:270)
    at executeDispatch (react-dom.development.js:561)
    at executeDispatchesInOrder (react-dom.development.js:583)
    at executeDispatchesAndRelease (react-dom.development.js:680)
    at executeDispatchesAndReleaseTopLevel (react-dom.development.js:688)
    at Array.forEach (<anonymous>)

**Comment by annabuczek** [comment]
I also experience an issue related to countryCodeEditable set to false. 

`TypeError: n.state.onlyCountries.find(...) is undefined`

It can be reproduced on a [demo page](https://bl00mber.github.io/react-phone-input-2.html) by setting values:
`{"onlyCountries": ["us", "ca"], "countryCodeEditable":false}`

**Comment by bizunas** [comment]
Firstly, thanks for this amazing component!

The fix fd76aa2 for 2.11.0 does not solve the problem. I can still recreate this issue with in demo page : 
    `{"country":"us","enableAreaCodes":false, "countryCodeEditable": false}`  

But 
    `{"country":"us","enableAreaCodes":true, "countryCodeEditable": false}` works as intended.

**Comment by bl00mber** [comment]
@bizunas this is not the fix for the 2.11.0, which is used on the demo page
